kombu/transport/kafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.kafka
```

Comment:
Can this be renamed kombu.transport.pykafka to avoid naming conflicts w/ other libraries (i.e., kafka-python) ?